islandora_video_streaming_panopto.module

```diff
@@ -1,5 +1,21 @@
 <?php
 
+/**
+ * Implements hook_menu().
+ */
+function islandora_video_streaming_panopto_menu() {
+  return array(
+    'admin/islandora/islandora_viewers/video_streaming_panopto' => array(
+      'title' => 'Panopto Video Streaming',
+      'description' => 'Configure the Kaltura Video Streaming Viewer',
```

Comment:
Copy and paste of Kaltura rather than Panopto here.

Reply:
Fixed.

Reply:
This appears to still be pending.